tools/ansible/tasks/wsu/main.yaml

```diff
@@ -46,7 +46,7 @@
       shell: |
         curl -s "{{ changelog.stdout }}" | \
         grep -A 1 "{{ kubernetes_version.stdout }}" | \
-        grep -A 1 node-windows | tail -n 1 | sed -e 's/.*<code>\(.*\)<\/code>.*/\1/'
+        grep -A 1 node-windows | tail -n 1 | sed -e 's/.*<code>\(.*\)<\/code>.*/\1/'| sed -e 's/.*<td>\(.*\)<\/td>.*/\1/'
```

Comment:
If `<code>` is not in the curl command, we should just use `grep -A 1 node-windows | tail -n 1 | sed -e 's/.*<td>\(.*\)<\/td>.*/\1/'`

Reply:
this is to ensure this works on 4.4, the releases before 1.18.3 contain <code>